### PR TITLE
Added WLS Plate functionality

### PR DESCRIPTION
**Pull request by MullenAustin**
Added WLS Plate functionality into ratpac.
WLS Plate generation is handled through the WatchmanWLSPSquareDetectorFactory.  The plates themselves and their reflectors are generated using WLSPFactory and WLSPCoverFactory.  Includes the macro mac/WLSP_visualize.mac to demonstrate usage.  Adds data folder data/Watchman_WLSP which contains the geo file and new WLS plate optical and material properties.

### CODE REVIEW

File: src/geo/src/DetectorConstruction.cc [hunk 1]
```diff
@@ -17,6 +17,8 @@
 #include <RAT/DetectorFactory.hh>
 #include <RAT/WatchmanDetectorFactory.hh>
 #include <RAT/TheiaDetectorFactory.hh>
+#include <RAT/WatchmanWLSPDetectorFactory.hh>
```
Comment (tgrbrooks) [inline src/geo/src/DetectorConstruction.cc:20]: File `WatchmanWLSPDetectorFactory.hh` not in this PR

Comment (MullenAustin) [reply src/geo/src/DetectorConstruction.cc]: Apologies for that, that was from an outdated factory that generated circular plates.  Forgot to remove that line before committing, should be resolved in the most recent commit.

**Review by tgrbrooks — APPROVED** [review]
Looks great

**Review by tgrbrooks — APPROVED** [review]
Code now compiles and performs as before.

**Review by tgrbrooks — APPROVED** [review]
(Testing to see if my review is accepted)

**Review by tgrbrooks — APPROVED** [review]
Trying one last time

**Pull request merged**